workbench_reviewer.module

```diff
@@ -66,3 +81,11 @@ function workbench_reviewer_preprocess_page(&$variables) {
     $variables['#attached']['library'][] = 'workbench_reviewer/workbench_reviewer';
   }
 }
+
+function workbench_reviewer_entity_field_access($operation, FieldDefinitionInterface $field_definition, AccountInterface $account, FieldItemListInterface $items = NULL) {
+  if ($field_definition->getName() == 'workbench_reviewer' && $operation == 'edit') {
```

Comment:
It would be safer to make `worbench_reviewer` a constant, something like `const WORKBENCH_REVIEWER_FIELD = 'workbench_reviewer';` That makes it easier to track (similar to how SASS uses vars.